src/crypto/CHIPCryptoPAL.h

```diff
@@ -116,6 +117,30 @@ inline constexpr char kPIDPrefixForCNEncoding[]    = "Mpid:";
 inline constexpr size_t kVIDandPIDHexLength        = sizeof(uint16_t) * 2;
 inline constexpr size_t kMax_CommonNameAttr_Length = 64;
 
+inline constexpr uint8_t kFabricBindingVersionV1 = 1u;
+
+inline constexpr size_t kVendorIdVerificationClientChallengeSize = 32u;
+
+// VIDVerificationStatement := statement_version || vid_verification_signer_skid || vid_verification_statement_signature
+inline constexpr size_t kVendorIdVerificationStatementV1MaxSize =
```

Comment:
Nit: Should we drop `Max` from these names ? afaik these are the _exact_ sizes.

Reply:
Fir some of them it's max, some it's exact. I will clarify.

Reply:
Done